packages/vhd-lib/src/Vhd/VhdAbstract.js

```diff
@@ -204,4 +213,93 @@ export class VhdAbstract {
     const relativePathToTarget = path.relative(aliasDir, path.resolve('/', targetPath))
     await handler.writeFile(aliasPath, relativePathToTarget)
   }
+
+  stream() {
+    const { header, footer, batSize } = this
+    const rawFooter = fuFooter.pack(footer)
+    checksumStruct(rawFooter, fuFooter)
+
+    // compute parent locator place and size
+    // update them in header
+    // update checksum in header
+
+    let offset = FOOTER_SIZE + HEADER_SIZE + batSize
+    for (let i = 0; i < PARENT_LOCATOR_ENTRIES; i++) {
+      if (header.parentLocatorEntry[i].platformDataSpace > 0) {
+        header.parentLocatorEntry[i].platformDataOffset = offset
+        offset += header.parentLocatorEntry[i].platformDataSpace
+      }
```

Comment:
Use variable to make code shorter and easier to edit:
```suggestion
      const entry = header.parentLocatorEntry[i]
      if (entry.platformDataSpace > 0) {
        entry.platformDataOffset = offset
        offset += entry.platformDataSpace
      }
```